### PR TITLE
Miami aces update

### PR DESCRIPTION
**Pull request by cristinabaldor**
**Code checks:**
1) Is your branch up to date with `main`? Update from `main` and resolve and merge conflicts before submitting.
2) Are you `JOIN`-ing to a subquery? Refactor as a `CTE`.
3) Do your CTEs significantly transform the data, or could they be refactored into simple `JOIN`s?
4) Will every `SELECT` column be used downstream? Remove superfluous columns.
5) Does every table `JOIN` introduce columns that are used downstream? Remove superfluous `JOIN`s.
6) Double check that your SQL conforms to the style guide.
   * All tables should be referenced in three-parts: `{database}.{schema}.{table}`
   * All columns sould be prefixed with a table alias: `t.column_name`
   * All keywords should be UPPERCASE; all identifiers should be `snake_case`
   * In the event an identifier shares a name with a keyword, surround it with [square brackets].
   * Spaces, not tabs.
    
**What is the purpose of this view?**
> *[extract|feed|clean-up|other] Brief explanation...*
Adding education level to the ACES view, and limiting to active employees or terminated this year.

### CODE REVIEW

File: extracts/v_extracts.mdcps_aces_survey.sql [hunk 1]
```diff
@@ -1,35 +1,37 @@
...
-      ,sr.termination_date
-FROM gabby.people.staff_roster sr
+
+FROM gabby.people.staff_roster s
 LEFT JOIN gabby.adp.workers_custom_field_group_wide_static cf
-  ON sr.associate_id = cf.worker_id
-WHERE sr.business_unit = 'KIPP Miami'
+  ON s.associate_id = cf.worker_id
+WHERE s.business_unit = 'KIPP Miami'
+  AND (termination_date >'2021-06-30' OR termination_date IS NULL)
```
Comment (cbini) [inline extracts/v_extracts.mdcps_aces_survey.sql:36]: should this be 2022? either way use `DATEFROMPARTS()` and `GLOBAL_ACADEMIC_YEAR()`

File: extracts/v_extracts.mdcps_aces_survey.sql [hunk 2]
```diff
@@ -1,35 +1,37 @@
 USE gabby
 GO
 
-CREATE OR ALTER VIEW extracts.mdcps_aces_survey AS
+--CREATE OR ALTER VIEW extracts.mdcps_aces_survey AS
```
Comment (cbini) [inline extracts/v_extracts.mdcps_aces_survey.sql:4]: uncomment

**Review by cbini — CHANGES_REQUESTED** [review]

File: extracts/v_extracts.mdcps_aces_survey.sql (hunk 1 above)
Comment (cristinabaldor) [reply extracts/v_extracts.mdcps_aces_survey.sql:36]: what i'm trying to get is only people who have null termination dates or termination dates within the academic year... is there a way to look for dates between the global academic year start and end dates?

Comment (cbini) [reply extracts/v_extracts.mdcps_aces_survey.sql:36]: `DATEFROMPARTS()` can do that, you feed it `GLOBAL_ACADEMIC_YEAR` as the year

**Review by cbini — APPROVED** [review]

**Pull request merged**